web/templates/web/admin_panel.html

```diff
@@ -104,6 +104,18 @@ <h1>{% trans "Admin panel" %}</h1>
                     </a>
                 </div>
             {% endif %}
+            {% if user|has_any_faq_permissions %}
+                <div class="ui yellow card">
+                    <a href="{% url 'FAQ-admin' %}" class="content">
+                        <div class="center aligned header">
+                            <i class="ui grey tools icon"></i>{% trans "FAQ" %}
```

Comment:
Replace tools icon with a suitable one for FAQ, perhaps a question mark or something similar?